.ebextensions/01_rails_db_setup.config

```diff
@@ -0,0 +1,10 @@
+files:
+  "/opt/elasticbeanstalk/hooks/appdeploy/post/01_rails_db_setup.sh":
+    mode: "000755"
+    owner: root
+    group: root
+    content: >
+      #!/usr/bin/env bash
+      cd /var/app/current
+      DISABLE_DATABASE_ENVIRONMENT_CHECK=1 bin/rails db:setup'
+      rm -- "$0"
```

Comment:
On the advice of [this article](https://blog.corpinfo.com/appdeploy-in-elastic-beanstalk), this last line removes the script file after running the previous commands.